File: Codesnippet/Codesnippet_php/CodeSnippet.php
<?php

use Adyen\Model\Checkout\Amount;
use Adyen\Model\Checkout\CreateCheckoutSessionRequest;
use Adyen\Service\Checkout\PaymentsApi;

// Include your idempotency key when you make an API request.
$requestOptions['idempotencyKey'] = "YOUR_IDEMPOTENCY_KEY";

// Set up the client and service.
$client = new \Adyen\Client();
$client->setXApiKey('YOUR_API_KEY');
$client->setEnvironment(\Adyen\Environment::TEST);
$client->setApplicationName('Test Application');//need this?
$client->setTimeout(30);//need this?

$service = new PaymentsApi($client);

// Creating Amount Object
$amount = new Amount();
$amount
    ->setValue(1500)
    ->setCurrency("EUR");

// Create Request
$sessionRequest = new CreateCheckoutSessionRequest();
$sessionRequest
    ->setMerchantAccount("YOUR_MERCHANT_ACCOUNT")
    ->setAmount($amount)
    ->setReference("YOUR_PAYMENT_REFERENCE")
    ->setReturnUrl("https://your-company.com/...");
    ->setCountryCode("NL");

$result = $service->sessions($sessionRequest);

File: Codesnippet/Codesnippet_java/CodeSnippet.java
//import the required class
package com.adyen;

import com.adyen.Client;
import com.adyen.service.checkout.PaymentsApi;
import com.adyen.model.checkout.Amount;
import com.adyen.model.checkout.CreateCheckoutSessionRequest;
import com.adyen.model.checkout.CreateCheckoutSessionResponse;
import com.adyen.enums.Environment;

//Setup Client and Service
Client client = new Client("YOUR_API_KEY", Environment.TEST);
//idempotency key?

//create session request
CreateCheckoutSessionRequest sessionRequest = new CreateCheckoutSessionRequest();
PaymentsApi checkoutPaymentsApi = new PaymentsApi(client);
Amount amount = new Amount().currency("EUR").value(1000L);
sessionRequest.setAmount(amount);
sessionRequest.setMerchantAccount("YOUR_MERCHANT_ACCOUNT");
sessionRequest.setReturnUrl("https://your-company.com/checkout?shopperOrder=12xy...");
sessionRequest.setReference("YOUR_PAYMENT_REFERENCE");
sessionRequest.setCountryCode("NL");
CreateCheckoutSessionResponse createCheckoutSessionResponse = checkoutPaymentsApi.sessions(sessionRequest);
